.github/workflows/auto_release.yml

```diff
@@ -0,0 +1,52 @@
+name: Release automatically
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+    paths:
+      - 'attractions/package.json'
+
+jobs:
+  release-from-pr:
+    name: Release from a PR that is marked with [new release]
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'new release')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Fetch the version number 📝
+        id: fetch-version
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          result-encoding: string
+          script:
+            const package = JSON.parse(require('fs').readFileSync('attractions/package.json'));
+            return `v${package.version}`;
+
+      - name: Fetch the changelog 📝
+        id: fetch-changelog
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          result-encoding: string
+          script:
+            const changelog = require('fs').readFileSync('CHANGELOG.md').split('\n');
+            const start = changelog.findIndex(line => line.startsWith('## '));
```

Comment:
Wouldn't it be more robust if you use the version returned from the previous step?

Reply:
It's just a convenience, no big deal. Don't wanna over-engineer it